scripts/superdesk-authoring/views/authoring-header.html

```diff
@@ -138,10 +139,10 @@
             </div>
           </div>
         </div>
-
-        <div class="authoring-header__row" ng-if="metadata.subjectcodes && schema.subject">
+
+        <div class="authoring-header__row" ng-if="metadata.subjectcodes && schema.subject && !metadata.subjects">

```

Comment:
This is an alternative list of subjects code for ntb. If it is defined, the current one will not be showed.